app/helpers/works_helper.rb

```diff
@@ -82,12 +82,12 @@ def marked_for_later?(work)
     reading && reading.toread?
   end
 
-  def mark_as_read_link(work)
-    link_to ts("Mark as Read"), mark_as_read_work_path(work)
+  def mark_as_read_button(work)
+    button_to t(".mark_as_read"), mark_as_read_work_path(work), method: :delete
```

Comment:
Because we don't change the text of these buttons based on context, it's probably better to skip lazy lookup here and use `works_helper.mark_as_read_button.mark_as_read` and `works_helper.mark_for_later_button.mark_for_later` instead. That way, the button will always use the text provided in `helpers/en.yml` and you won't need to provide text and `i18n-use` comments every time it's used in a view file.

(I also think you'd be fine with shorter keys, like `works_helper.mark_for_later_button.text` or even just `works_helper.mark_for_later_button`.)